src/main/java/org/powerflows/dmn/engine/configuration/DecisionEngineConfiguration.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2018-present PowerFlows.org - all rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.powerflows.dmn.engine.configuration;
+
+
+import org.powerflows.dmn.engine.DecisionEngine;
+import org.powerflows.dmn.engine.reader.DecisionReader;
+
+public interface DecisionEngineConfiguration {
+
+    void setDecisionReader(DecisionReader decisionReader);
```

Comment:
It shouldn't use reader directly, there should be something like decision provider or source that would provide Decisions...